Documentation/BareMetalInstallation.md

```diff
@@ -14,7 +14,7 @@ You must be willing to wipe your disk's contents to allow for writing the Sereni
 
 Serenity currently has no support for USB but some machines will emulate PS/2 keyboards and mice in the BIOS via USB. BIOS USB PS/2 emulation can be buggy so having real PS/2 input devices is recommended. A minimum of 128 MB RAM and a Pentium III class CPU are required.
 
-At present there is no real GPU support so don't expect OpenGL, Vulkan nor accelerated video playback and encoding support. Serenity currently relies upon VESA BIOS extensions to provide its display output and so it only runs on BIOS-based PCs. There is no WiFi support and the only three network card chipsets are currently supported: Novell NE2000 and Intel e1000. The e1000 driver has only been tested with qemu and VirtualBox although it may work with NICs such as those using the Intel 82545XX, 82540XX, 82546XX or similar chipsets. The sole sound card supported is the SoundBlaster 16 ISA.
+At present there is no real GPU support so don't expect OpenGL, Vulkan nor accelerated video playback and encoding support. Serenity currently relies upon VESA BIOS extensions to provide its display output and so it only runs on BIOS-based PCs. There is no WiFi support and the only three network card chipsets are currently supported: Intel e1000. The e1000 driver has only been tested with qemu and VirtualBox although it may work with NICs such as those using the Intel 82545XX, 82540XX, 82546XX or similar chipsets. The sole sound card supported is the SoundBlaster 16 ISA.
```

Comment:
>There is no WiFi support and the only three network card chipsets are currently supported: Intel e1000.

Minor edit: "There is no WiFi support, and only one network card chipset is currently supported: Intel e1000."

Reply:
There's at least one outdated information in the same paragraph:
> The sole sound card supported is the SoundBlaster 16 ISA.

Maybe it would be a better idea to remove references to (non)-supported devices in this document and refer to the Documentation/HardwareCompatibility.md document instead of duplicating information here.